webrtc.html

```diff
@@ -5869,25 +5869,20 @@ <h2>Methods</h2>
                       <p>Run the following steps in parallel:</p>
                       <ol>
                         <li>
-                          <p>Determine if negotiation would be needed in order to
-                          replace <var>sender</var>'s existing track with
-                          <var>withTrack</var>. Ignore whether the sender already
-                          needs negotiation, in this determination.
-                          Negotiation is not needed if <var>withTrack</var>
-                          is null. Ignore which <code>MediaStream</code> the track
-                          resides in and the <code>id</code> attribute of the track
-                          in this determination. Also, ignore
-                          <var>transceiver</var>'s <a>[[\Direction]]</a> slot in
-                          this determination. If negotiation is needed, then
-                          <a>reject</a> <var>p</var> with a newly
+                          If <var>withTrack</var> is not <code>null</code> and
+                          <var>transceiver</var> is already associated with a
+                          media description, determine if <var>withTrack</var>
+                          can be sent immediately by the sender without violating
+                          the sender's already-negotiated envelope, and if it
```

Comment:
Is "already-negotiated envelope" defined well enough?

Reply:
This also loses the "ignore direction" part; if the direction is "recvonly", the track can't immediately be sent, so should the promise be rejected?

Reply:
@taylor-b The new text also ignores direction. We want replaceTrack to succeed whenever it is valid to replace the track, so in the "recvonly" case it should be fine to replace the track regardless of the envelope, is that right?

Reply:
@jan-ivar will bring back "ignore direction"